content/core-features/authentication.textile

```diff
@@ -453,14 +453,16 @@ h4(#target-specifiers). Target specifiers
 
 A revocation request must include one or more target specifiers which define the token(s) that are affected by the request. Where multiple target specifiers are included, this is equivalent to making multiple independent revocation requests, each for a single target. The tokens affected by a request are those that match at least one of the target specifiers included in the request.
 
+All target specifiers are formatted as: @key:<string>@. 
+
 By default, supported target specifiers are:
 
-- clientId:<clientId> := For a string @clientId@, matches tokens that have a non-empty @clientId@ equal to that given.
-- revocationKey:<revocationKey> := Matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@.
+- clientId := Key that takes a string value, matches tokens that have a non-empty @clientId@ equal to that given, for example: <code>targets: ["clientId:client1@example.com" ]</code>
+- revocationKey := Key that takes a string value, matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@, for example: <code>targets: ["revocationKey:users.group1@example.com"]</code>
```

Comment:
> Key that takes a string value, matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@,

A little confusing. Perhaps reword along similar lines to the previous suggestion.